lti_consumer/views.py

```diff
@@ -0,0 +1,97 @@
+from rest_framework import viewsets, status
+from rest_framework.decorators import action
+from rest_framework.response import Response
+from django_filters.rest_framework import DjangoFilterBackend
+
+from lti_consumer.models import LtiAgsLineItem
+from lti_consumer.serializers import (
+    LtiAgsLineItemSerializer,
+    LtiAgsScoreSerializer,
+)
+
+from lti_consumer.lti_1p3.extensions.rest_framework.permissions import LtiAgsPermissions
+from lti_consumer.lti_1p3.extensions.rest_framework.authentication import Lti1p3ApiAuthentication
+from lti_consumer.lti_1p3.extensions.rest_framework.renderers import (
+    LineItemsRenderer,
+    LineItemRenderer,
+)
+from lti_consumer.lti_1p3.extensions.rest_framework.parsers import (
+    LineItemParser,
+    LineItemScoreParser,
+)
+
+
+class LtiAgsLineItemViewset(viewsets.ModelViewSet):
+    """
+    A simple ViewSet for viewing and editing accounts.
+    """
+    serializer_class = LtiAgsLineItemSerializer
+    pagination_class = None
+
+    # Custom permission classes for LTI APIs
+    authentication_classes = [Lti1p3ApiAuthentication]
+    permission_classes = [LtiAgsPermissions]
+
+    # Renderer/parser classes to accept LTI AGS content types
+    renderer_classes = [
```

Comment:
@pcockwell Both the results and scores service return different content types:
See https://www.imsglobal.org/spec/lti-ags/v2p0#media-type-and-schema and https://www.imsglobal.org/spec/lti-ags/v2p0#media-type-and-schema.

You can either build new renderer classes, or modify the existing ones to return the proper `Content-Types` depending on the request.